UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/DoubleHandController.cs

```diff
@@ -24,12 +24,26 @@ public class DoubleHandController : MonoBehaviour
 	public HandsController RelatedHandsController;
 
 
-	public void HideHands(bool HideState)
+	//0 - Hide both hands, 1 - hide left hand, 2 - hide right hand
```

Comment:
Please use enums in such cases, I don't see any reasons why these values should be ints

Reply:
good enough?

Reply:
Better, but I still see many places where you still use ints